bitnami/common/templates/_labels.tpl

```diff
@@ -3,7 +3,12 @@
 Kubernetes standard labels
 */}}
 {{- define "common.labels.standard" -}}
+app: {{ include "common.names.name" . }}
 app.kubernetes.io/name: {{ include "common.names.name" . }}
+{{- if .Release.appVersion -}}
+app.kubernetes.io/version: {{ .Release.appVersion }}
+version: {{ .Release.appVersion }}
+{{- end -}}
```

Comment:
We should be extremely cautious when including labels that change everytime the version is bumped (`appVersion` in this case). In the past, this lead into upgradability issues.